lib/brains/tests/test_webhook.py

```diff
@@ -39,6 +66,42 @@ def test_get_ok(self):
     def test_post_missing(self):
         eq_(self.client.post(self.url, data={}).status_code, 422)
 
-    def test_post_ok(self):
+    @patch('lib.brains.webhooks.Processor.process')
+    def test_post_ok(self, process):
         self.mocks['notify'].parse.return_value = parsed()
         eq_(self.client.post(self.url, data=example()).status_code, 204)
+        ok_(process.called)
+
+
+class TestWebhookSubscriptionCharged(BraintreeTest):
+    kind = 'subscription_charged_successfully'
+
+    def setUp(self):
+        # Create the world.
+        self.buyer, self.braintree_buyer = create_braintree_buyer()
+        self.method = create_method(self.braintree_buyer)
+        self.seller, self.seller_product = create_seller()
+        self.braintree_sub = BraintreeSubscription.objects.create(
+            paymethod=self.method,
+            seller_product=self.seller_product,
+            provider_id='some-bt:id'
+        )
+        self.sub = subscription
+
+    def test_ok(self):
+        Processor(parsed(kind=self.kind, subject=self.sub)).process()
+        eq_(Transaction.objects.get().status, constants.STATUS_CHECKED)
+
+    def test_sets_amount(self):
+        Processor(parsed(kind=self.kind, subject=self.sub)).process()
+        eq_(Transaction.objects.get().amount, Decimal('10'))
+
+    def test_sets_uid_support(self):
+        Processor(parsed(kind=self.kind, subject=self.sub)).process()
+        eq_(Transaction.objects.get().uid_support, 'bt:id')
```

Comment:
its just a string set up above in the same file, don't normally make vars for everything